app/javascript/shared/requestHelpers.js

```diff
@@ -28,4 +29,15 @@ const requestsFromPlates = function (plateObjs) {
   return requestsArray
 }
 
-export { requestIsActive, requestsFromPlates, requestIsLibraryCreation }
+const handleFailedRequest = function (request) {
+  // generate an alert on the page
+  const title = request.response.data.message[1]
+  const messages = request.response.data.message[0].join(', ')
+  eventBus.$emit('push-alert', {
```

Comment:
Without a state manager this seems like the best way to handle the cross-component error handling, nice 👍